libs/cardano-ledger-test/src/Test/Cardano/Ledger/Examples/ConwayFeatures.hs

```diff
@@ -539,10 +541,15 @@ testGov pf = do
     (utxo0, _) = utxoFromTestCaseData pf (proposal pf)
     committee = Committee @era (Map.fromList [(ccColdCred @era pf, EpochNo 100)]) (unsafeBoundRational (1 % 2))
     committeeState = CommitteeState (Map.fromList [(ccColdCred pf, Just (ccHotCred pf))])
+    initialEnactState =
+      def
+        & ensCurPParamsL .~ pp
+        & ensCommitteeL .~ SJust committee
+    initialDRepPulsingState = DRComplete def (def & rsEnactStateL .~ initialEnactState)
```

Comment:
This was a bit surprising to me. I had to think about this for a long time. I guess that one can explain it this way.
In the transiition to the initital epoch, the pre-initial pulser has to complete, and that needs to use the snap shot of the previous EnactState snapshot.  So perhaps the name 'initialEnactState' is not a good name? perhaps the 'initialEnactSnapshot'.  Of course I might just be confused.  Perhaps some comments.

Reply:
initialEnactState is a pretty good name, because it is NOT a snapshot. It is in fact the state that will be set at the next epoch in Conway. In other words it is the future EnactState. Whenever we running tests or whenever we enter Conway from Babbage, we must ensure EnactState is set to the same value in the DRepPulser (in DRComplete) and in the ConwayGovState, otherwise the value in ConwayGovState will get overwritten at the very next epoch